diff --git a/dist/cjs/cli/lib/localDeployment/run.js b/dist/cjs/cli/lib/localDeployment/run.js
index ded88ce2e5811437bb215b2a3b967c6e78d0ad5a..7044a6d8108ca069c6a9e1094de7e4c80d6e0311 100644
--- a/dist/cjs/cli/lib/localDeployment/run.js
+++ b/dist/cjs/cli/lib/localDeployment/run.js
@@ -141,7 +141,7 @@ async function runLocalBackend(ctx, args) {
   await ensureBackendRunning(ctx, {
     cloudPort: ports.cloud,
     deploymentName: args.deploymentName,
-    maxTimeSecs: 30
+    maxTimeSecs: 240
   });
   return {
     cleanupHandle
diff --git a/dist/cli.bundle.cjs b/dist/cli.bundle.cjs
index 92451cf6c944bdee00a79ec0c4a1c0226f9350f2..d50ec4eb82873bf6a6a5a1c144cdbd2c3a40c623 100644
--- a/dist/cli.bundle.cjs
+++ b/dist/cli.bundle.cjs
@@ -128207,7 +128207,7 @@ async function runLocalBackend(ctx, args) {
   await ensureBackendRunning(ctx, {
     cloudPort: ports.cloud,
     deploymentName: args.deploymentName,
-    maxTimeSecs: 30
+    maxTimeSecs: 240
   });
   return {
     cleanupHandle
diff --git a/dist/esm/cli/lib/localDeployment/run.js b/dist/esm/cli/lib/localDeployment/run.js
index fa691c57543366678db3a3d308e64df90bc2864e..f0b0c192dccaa4dc56fb4aacd26644da2de11006 100644
--- a/dist/esm/cli/lib/localDeployment/run.js
+++ b/dist/esm/cli/lib/localDeployment/run.js
@@ -104,7 +104,7 @@ export async function runLocalBackend(ctx, args) {
   await ensureBackendRunning(ctx, {
     cloudPort: ports.cloud,
     deploymentName: args.deploymentName,
-    maxTimeSecs: 30
+    maxTimeSecs: 240
   });
   return {
     cleanupHandle
